test/WebJobs.Script.Tests/Artifacts/ApiHub/ApiHubTestHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Sure, but then if everyone else will need test helpers specific to their extensions we'll end having a bunch of folders in the root. That is why I thought it would be good to put them in a separate folder. I called it Artifacts but if you want a different name I can change.